### ISSUE TITLE
Uncaught TypeError: Cannot read property 'getBufferPosition' of undefined

### ISSUE THREAD

**Issue by veryluckyxyz** [yasuyuky/autocomplete-clang#25]
Uncaught TypeError: Cannot read property 'getBufferPosition' of undefined

**Atom Version**: 0.161.0
**System**: Mac OS X 10.10.1
**Thrown From**: [autocomplete-clang](https://github.com/yasuyuky/autocomplete-clang) package, v0.5.2

### Steps To Reproduce

1. ...
2. ...

### Stack Trace

```
At /Applications/Atom.app/Contents/Resources/app/node_modules/text-buffer/lib/history.js:121

TypeError: Cannot read property 'getBufferPosition' of undefined
  at TextEditor.module.exports.TextEditor.getCursorBufferPosition (/Applications/Atom.app/Contents/Resources/app/src/text-editor.js:1300:34)
  at AutocompleteClangView.module.exports.AutocompleteClangView.handleChanged (/Users//.atom/packages/autocomplete-clang/lib/autocomplete-clang-view.coffee:65:19)
  at /Users//.atom/packages/autocomplete-clang/lib/autocomplete-clang-view.coffee:41:47
  at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:82:11)
  at TextBuffer.module.exports.TextBuffer.applyPatch (/Applications/Atom.app/Contents/Resources/app/node_modules/text-buffer/lib/text-buffer.js:590:20)
  at TextBuffer.module.exports.TextBuffer.setTextInRange (/Applications/Atom.app/Contents/Resources/app/node_modules/text-buffer/lib/text-buffer.js:503:12)
  at Selection.module.exports.Selection.insertText (/Applications/Atom.app/Contents/Resources/app/src/selection.js:446:43)
  at /Applications/Atom.app/Contents/Resources/app/src/text-editor.js:738:31
  at /Applications/Atom.app/Contents/Resources/app/src/text-editor.js:777:27
  at History.module.exports.History.transact (/Applications/Atom.app/Contents/Resources/app/node_modules/text-buffer/lib/history.js:112:18)
  at TextBuffer.module.exports.TextBuffer.transact (/Applications/Atom.app/Contents/Resources/app/node_modules/text-buffer/lib/text-buffer.js:683:27)
  at TextEditor.module.exports.TextEditor.transact (/Applications/Atom.app/Contents/Resources/app/src/text-editor.js:1146:26)
  at TextEditor.module.exports.TextEditor.mutateSelectedText (/Applications/Atom.app/Contents/Resources/app/src/text-editor.js:770:19)
  at TextEditor.module.exports.TextEditor.insertText (/Applications/Atom.app/Contents/Resources/app/src/text-editor.js:735:21)
  at TextEditor.object.(anonymous function) [as insertText] (/Applications/Atom.app/Contents/Resources/app/node_modules/underscore-plus/lib/underscore-plus.js:77:27)
  at TextEditor.module.exports.TextEditor.insertNewline (/Applications/Atom.app/Contents/Resources/app/src/text-editor.js:754:19)
  at TextEditor.object.(anonymous function) [as insertNewline] (/Applications/Atom.app/Contents/Resources/app/node_modules/underscore-plus/lib/underscore-plus.js:77:27)
  at TextEditor.atom.commands.add.stopEventPropagationAndGroupUndo.editor:newline (/Applications/Atom.app/Contents/Resources/app/src/text-editor-element.js:576:19)
  at /Applications/Atom.app/Contents/Resources/app/src/text-editor-element.js:291:34
  at History.module.exports.History.transact (/Applications/Atom.app/Contents/Resources/app/node_modules/text-buffer/lib/history.js:112:18)
  at TextBuffer.module.exports.TextBuffer.transact (/Applications/Atom.app/Contents/Resources/app/node_modules/text-buffer/lib/text-buffer.js:683:27)
  at TextEditor.module.exports.TextEditor.transact (/Applications/Atom.app/Contents/Resources/app/src/text-editor.js:1146:26)
  at atom-text-editor.newCommandListeners.(anonymous function) (/Applications/Atom.app/Contents/Resources/app/src/text-editor-element.js:290:22)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:243:29)
  at /Applications/Atom.app/Contents/Resources/app/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:549:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:391:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/Applications/Atom.app/Contents/Resources/app/src/window-event-handler.js:167:20)

```

/cc @atom/core

**Comment by hsidky** [comment]
This is a recurring problem for me too. I notice it happens when I have split pane view. It never occurs when I'm viewing one document at a time.

**Comment by yasuyuky** [comment]
Hi, thanks for your feedbacks.
This problem was caused by listening wrong event on wrong place.
It is fixed on the latest v0.6.0. Check it out.

If you still have the problem, reopen this issue.
Thanks.
